spec/fixtures/credential/key.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
Would you mind leaving this one file here for now? I'm going to need it in a change I'm working on right now, and I can move it out as part of that change.